api/db/migrate/20210902162826_create_spree_webhooks_tables.rb

```diff
@@ -0,0 +1,30 @@
+class CreateSpreeWebhooksTables < ActiveRecord::Migration[6.1]
+  def change
+    create_table :spree_webhooks_events do |t|
+      t.text :name, null: false
+      t.boolean :enabled, default: true
+
+      t.timestamps
+    end
+
+    create_table :spree_webhooks_endpoints do |t|
+      t.text :url, null: false
+      t.boolean :enabled, default: true
+
+      create_subscriptions_column!(t)
+
+      t.timestamps
+    end
+  end
+
+  private
+
+  def create_subscriptions_column!(table)
+    case ActiveRecord::Base.connection.adapter_name
+    when 'Mysql2'
+      table.json :subscriptions
+    when 'PostgreSQL'
+      table.jsonb :subscriptions, default: ['*']
```

Comment:
@sebastian-palma by default let's have this as null so no events will be emitted